### ISSUE TITLE
Review SerialisationTest.java

### ISSUE THREAD

**Issue by gedeck** [cdd/bioassay-template#74]
```
		// deserialise file to schema
		Schema schema2 = SchemaUtil.deserialise(tmpFile).schema;
		assertTrue("Schema does not match (destination file in JSON).", schemaFromJSON.equals(schema2));
		
		// deserialise without filename clue
		try (InputStream istr = new FileInputStream(tmpFile))
		{
			Schema schema3 = SchemaUtil.deserialise(istr).schema;
			assertTrue("Schema stream does not match (destination file in JSON).", schemaFromJSON.equals(schema2));
		}
```
Should the last assert statement test schema3?
```
			assertTrue("Schema stream does not match (destination file in JSON).", schemaFromJSON.equals(schema3));
````

**Comment by kodecharlie** [comment]
Yes, I'm pretty sure there are 2 places in `SerialisationTest.java` where `schema2` should be `schema3` instead.

**Comment by gedeck** [comment]
Closed with #81
